Dima/Classic_Algorithms/collatz_conjecture/collatz_conjecture.py

```diff
@@ -0,0 +1,31 @@
+from Dima.Decorators.time_decorator import timer
+
+
+@timer
+def collatz_conjecture(number):
+    try:
+        number = int(number)
+    except ValueError:
+        ValueError('ValueError')
```

Comment:
Please take a look on comments for other tasks regarding exceptions
Change messages to ValueError_1 and ValuerError_2 + tests